src/account/kernel/api/installModule.ts

```diff
@@ -17,12 +17,14 @@ export const installModule = ({
 }: {
   client: PublicClient
   account: Account
-  module: Module
+  module: KernelModule
```

Comment:
do you think this makes it more complex for the dev? what if we just had one module typescript type and accounts that dont support certain a `module.type` just throw an error when using them?

Reply:
@kopy-kat that would work but it is not a type safe .. since typescript won't complain about the type but during execution it will fail

Reply:
@kopy-kat I think that would improve dev experience since it will be clear for dev which modules are supported for accounts they build for .. wdyt?